### PR TITLE
update README.md

### PR DESCRIPTION
**Pull request by beliaev-maksim**
* issue #58 is closed as completed, remove the section from the doc.
* minor fixed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,15 +40,10 @@ $ juju relate kube-state-metrics prometheus-k8s
 In a similar manner any charm that exposes a scrape target may be
 related to the Prometheus charm.
 
-At present it is expected that all relations the Prometheus Operator
-partakes in are within the same Juju model.
-[Further development](https://github.com/canonical/prometheus-operator/issues/58)
-may extend this to allow cross model scrape targets.
```
Comment (simskij) [inline README.md:47]: This is still the case. Prometheus might in some cases (like same-cluster, different models) be able to scrape across model boundaries, but usually what we recommend is for users to set up a grafana agent in the remote model and use `remote_write` to get their metrics into Prometheus.

The reference to the issue can go, however, as we're not expecting to change this behavior.

**Review by simskij — CHANGES_REQUESTED** [review]
Thanks Maksim! I've added a small clarification that I think we should address before merging. 👍🏼

File: README.md (hunk 1 above)
Comment (beliaev-maksim) [reply README.md:47]: @simskij 
what is about adding such a note?


> Note: At present it is expected that all relations the Prometheus Operator 
> partakes in are within the same Juju model. For alternative, please set up a 
> grafana agent in the remote model and use `remote_write` to get metrics into Prometheus.

**Review by Abuelodelanada — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (simskij) [reply README.md:47]: > At present it is expected that all relations that Prometheus are part of are within the same Juju model. For cross-model relations, please set up a Grafana Agent in the remote model and use remote_write to get the metrics into Prometheus.

some minor grammar changes, but all in all - looks good! 👍🏼

Comment (beliaev-maksim) [reply README.md:47]: done

File: README.md [hunk 2]
```diff
@@ -41,15 +40,15 @@ $ juju relate kube-state-metrics prometheus-k8s
...
+> Note: At present it is expected that all relations that Prometheus are
```
Comment (sed-i) [inline README.md:43]:
```suggestion
> Note: At present it is expected that all relations that Prometheus is
```

**Review by sed-i — APPROVED** [review]

**Review by simskij — APPROVED** [review]

**Pull request merged**